oidc-api/src/main/java/uk/gov/di/authentication/oidc/lambda/OrchFrontendAuthorizerHandler.java

```diff
@@ -0,0 +1,87 @@
+package uk.gov.di.authentication.oidc.lambda;
+
+import com.amazonaws.services.lambda.runtime.Context;
+import com.amazonaws.services.lambda.runtime.RequestHandler;
+import com.amazonaws.services.lambda.runtime.events.APIGatewayProxyRequestEvent;
+import org.apache.commons.net.util.SubnetUtils;
+import org.apache.logging.log4j.LogManager;
+import org.apache.logging.log4j.Logger;
+import org.apache.logging.log4j.ThreadContext;
+import uk.gov.di.orchestration.shared.entity.AuthPolicy;
+import uk.gov.di.orchestration.shared.services.ConfigurationService;
+
+import static uk.gov.di.orchestration.shared.helpers.InstrumentationHelper.segmentedFunctionCall;
+
+public class OrchFrontendAuthorizerHandler
+        implements RequestHandler<APIGatewayProxyRequestEvent, AuthPolicy> {
+
+    private static final Logger LOG = LogManager.getLogger(OrchFrontendAuthorizerHandler.class);
+
+    private final ConfigurationService configurationService;
+
+    private final String[] validIps = {
```

Comment:
can we add a link to `https://sites.google.com/a/digital.cabinet-office.gov.uk/gds/working-at-gds/gds-internal-it/gds-internal-it-network-public-ip-addresses` in a comment please?

Reply:
I think there are some we can get rid of from this list too